packages/react-scripts/scripts/build.js

```diff
@@ -40,7 +40,36 @@ const configFactory = require('../config/webpack.config');
 const paths = require('../config/paths');
 const checkRequiredFiles = require('react-dev-utils/checkRequiredFiles');
 const formatWebpackMessages = require('react-dev-utils/formatWebpackMessages');
-const printHostingInstructions = require('react-dev-utils/printHostingInstructions');
+
+//
+// Override the default hosting instructions with FTW specific instructions.
+//
+// const printHostingInstructions = require('react-dev-utils/printHostingInstructions');
+//
+function printHostingInstructions(
+  appPackage,
+  publicUrl,
+  publicPath,
+  buildFolder,
+  useYarn
+) {
+  console.log();
+  console.log(`The ${chalk.cyan(buildFolder)} folder is ready to be deployed.`);
+  console.log();
+  console.log('You may serve it by running:');
+  console.log();
+  console.log(`  ${chalk.cyan('yarn')} start`);
```

Comment:
Maybe "yarn run dev-server" instead?

"yarn start" is used by host service which should have NODE_ENV & PORT set in environment variables. (PORT is handled by host services pretty independently)

(So, this should work too: `NODE_ENV=development PORT=3000 yarn start`)

Reply:
@Gnito I copied the "yarn start" from this article (https://www.sharetribe.com/docs/ftw-hosting/how-to-deploy-ftw-to-production/#building-the-app) which is also linked below.

What I'm thinking is that isn't the "yarn build" command meant to build the production bundle which is meant to be run in a production mode mainly? What would be the reason to run the build bundle in dev mode?

So I'm wondering that one option would be to change the instructions from "You may serve it by running:" to "You may serve it on production mode by running:", and then the command would be `NODE_ENV=production PORT=3000 yarn start`. Would that make sense?

Reply:
Yes, that makes sense. My main point is that those 2 environment variables need to be handled in some way.

NODE_ENV is used to [take proper dotenv files into use](https://github.com/sharetribe/create-react-app/blob/master/packages/react-scripts/config/env.js#L21).
PORT is something that host services usually want to handle on their own (3000 is fine for local machine)